src/tests/SearchView.test.jsx

```diff
@@ -9,8 +9,14 @@ import userEvent from '@testing-library/user-event';
 import SearchView from '../components/Search/SearchView';
 import { render, screen, waitFor, store } from './utils/test-utils';
 
-const { getByLabelText, getByRole, getByText, queryByRole, queryByText } =
-  screen;
+const {
+  findByRole,
+  getByLabelText,
+  getByRole,
+  getByText,
+  queryByRole,
+  queryByText,
+} = screen;
```

Comment:
That's really up to you, but I find it easier to just call `screen.getByText` (and friends) directly in the test code. The main rationale is that when you have to use a different method that's not in the declaration yet, you don't have to add the new one, and possibly remove the old one if it's not used anymore.